config/test.js

```diff
@@ -1,7 +1,8 @@
 // Put any test configuration overrides here.
 module.exports = {
   appName: 'test-app-set-from-config',
-  apiHost: 'https://localhost',
+  // If you see a test make a request to this, it's probably a bug.
+  apiHost: 'http://your-test-is-touching-the-api-pls-stop',
```

Comment:
This is cool but can we expand "pls" to "please"? 😄

Reply:
I'll just link to this http://time.com/4373616/text-abbreviations-acronyms/ . Only kidding.

Reply:
Ha, 'pls' is actually in there.

Reply:
😅